ext/pdo_dblib/tests/batch_stmt_ins_exec.phpt

```diff
@@ -0,0 +1,61 @@
+--TEST--
+ PDO_DBLIB driver supports a batch of queries containing select, insert, update, exec statements
+ this test will fail when using the dblib driver that hasn't been patched
+--SKIPIF--
+<?php
+if (!extension_loaded('pdo_dblib')) die('skip not loaded');
+require dirname(__FILE__) . '/config.inc';
+?>
+--FILE--
+<?php
+require dirname(__FILE__) . '/config.inc';
+
+// creating a proc need to be a statement in it's own batch, so we need to do a little setup first
+$db->query("create table #wf_pdo(id int); ");
```

Comment:
WF prefix is going to be confusing. Other tests use more generic table names like php_test. Anything wrong with that?